tests/components/homeworks/test_config_flow.py

```diff
@@ -331,7 +341,10 @@ async def test_reconfigure_flow_flow_duplicate(
 
 
 async def test_reconfigure_flow_flow_no_change(
-    hass: HomeAssistant, mock_config_entry: MockConfigEntry, mock_homeworks: MagicMock
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+    mock_homeworks: MagicMock,
+    mock_async_schedule_reload: MagicMock,
 ) -> None:
     """Test reconfigure flow."""
```

Comment:
I think this one needs `reload_on_entry_update=False`

Reply:
done